packages/graphql/src/modules/user/app/authenticateFacebookUser.ts

```diff
@@ -49,7 +49,8 @@ const authenticateFacebookUser = ({
   if (existingUser) {
     // verify if name was updated
     if (name.length !== existingUser.name.length) {
-      await userRepository.updateUser({id: existingUser.id, proprieties: {name}});
+      existingUser.name = name;
```

Comment:
I think in this case it's not a good practice to change an object directly. Is there a chance this causes side effects? How about creating a copy of this object spreading the existing properties and changing the name?

```
const targetUser = { ...existingUser, name };
await userRepository.update(targetUser)
```